xml/deployment_sysreqs.xml

```diff
@@ -781,6 +781,250 @@
   </para>
  </sect1>
 
+ <sect1 xml:id="sec.deploy.requirements.public_cloud">
+  <title>General Public Cloud Requirements</title>
+  <para>
+   In Amazon EC2, the adminstrative instance must be launched with an IAM role
+   that allows full access to the EC2 API. In Microsoft Azure, all security
+   credentials will be collected during setup. In Google Compute Engine, the
+   instance must be launched with an IAM role including <literal>Compute
+   Admin</literal> and <literal>Service Account Actor</literal> scopes.
+  </para>
+  <para>
+   Select an instance size for the Administrative node that meets the system
+   requirements as documented in the &productname; documentation.
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term>Memory</term>
+    <listitem>
+     <para>
+      Minimal main memory: 8GB
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Volume Size</term>
+    <listitem>
+     <para>
+      Root volume size should be at least 40GB and is dependent on the size and
+      number of different containers running in the cluster. The default root
+      volume size of the images is smaller than 40GB in some Public Cloud
+      frameworks; you must resize the root volume to meet your needs upon
+      instance launch using the command line tools or the web interface for the
+      framework of your choice.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect1>
+
+ <sect1 xml:id="sec.deploy.requirements.public_cloud_network">
+  <title>Public Cloud Network Requirements</title>
```

Comment:
Can you have a look how much of this can be merged with the networking requirements? 
I would opt for an additional subsection in networking for only the public cloud specific things. The rest reads like a lot of duplication of general requirements. If they are not described in the general reqs move them there.